File: src/main/java/com/aquasecurity/plugins/trivy/model/DataSource.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class DataSource {
    @JsonProperty("ID")
    public String iD;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("URL")
    public String uRL;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/oss/DataSource.kt
package com.aquasecurity.plugins.trivy.model.oss

import com.fasterxml.jackson.annotation.JsonProperty

class DataSource {
  @JsonProperty("ID") var iD: String? = null

  @JsonProperty("Name") var name: String? = null

  @JsonProperty("URL") var uRL: String? = null
}
